aws/resource_aws_launch_template.go

```diff
@@ -549,6 +554,10 @@ func resourceAwsLaunchTemplateRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("user_data", ltData.UserData)
 	d.Set("vpc_security_group_ids", aws.StringValueSlice(ltData.SecurityGroupIds))
 
+	if ltData.EbsOptimized != nil {
```

Comment:
We should probably additionally call `d.Set("ebs_optimized", "")` before this `if` to handle ensure Terraform catches the scenario where the attribute is set to "unspecified" out of band.